lib/trino-geospatial-toolkit/src/main/java/io/trino/geospatial/GeometryUtils.java

```diff
@@ -187,48 +176,13 @@ public static boolean isPointOrRectangle(OGCGeometry ogcGeometry, Envelope envel
     public static org.locationtech.jts.geom.Geometry jtsGeometryFromJson(String json)
     {
         try {
-            org.locationtech.jts.geom.Geometry emptyGeoJsonOverride = getEmptyGeometryOverride(json);
-            if (emptyGeoJsonOverride != null) {
-                return emptyGeoJsonOverride;
-            }
             return new GeoJsonReader().read(json);
         }
         catch (ParseException | IllegalArgumentException e) {
```

Comment:
What code throws `IllegalArgumentException`? javadoc of `GeoJsonReader#read(java.lang.String)` mentions only `ParseException`?